core/variant/variant.h

```diff
@@ -91,11 +94,14 @@ class Variant {
 		VECTOR3,
 		VECTOR3I,
 		TRANSFORM2D,
+		VECTOR4,
```

Comment:
PackedVector4Array?

Reply:
you can still use a regular array for this use case or PackedColorArray, so its not suuuuper important.

Reply:
Is there explicit reasoning to not have Vector4 array but still have Vector2+Vector3 array? If not, we should either add PackedVector4Array or remove Vector2+Vector3 arrays.